Cargo.toml

```diff
@@ -18,13 +18,14 @@ features = ["cookies", "json"]
 status = "actively-developed"
 
 [features]
-default = ["http2", "static-curl", "text-decoding"]
+default = ["http2", "static-curl", "static-ssl", "text-decoding"]
```

Comment:
Adding this to the default features would be a change in behavior. I think leaving it as an optional feature would be the best option for now.